ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Xcode/tasks/main.yml

```
@@ -82,3 +82,6 @@
     XCODE_INSTALL_PASSWORD: "{{ Apple_ID_Password }}"
     FASTLANE_SESSION: "{{ FASTLANE_SESSION }}"
     FASTLANE_DONT_STORE_PASSWORD: 1
+
+- name: Set Xcode switch path to "/Applications/Xcode.app"
+  shell: sudo xcode-select --switch "/Applications/Xcode.app"
```

Ansible playbooks will be running as root anyway, so don't need `sudo`
```suggestion
  shell: xcode-select --switch "/Applications/Xcode.app"
```

---

Not on MacOS. They run as Admin(istrator), which will require sudo. They run as Admin from AWX and from my own machine as I do not have root access to some (perhaps all) macos machines